QuantConnect.TDAmeritrade/TDAmeritradeBrokerage.cs

```diff
@@ -284,15 +251,19 @@ public override List<Holding> GetAccountHoldings()
 
             var holdings = new List<Holding>(positions.Count);
 
+            var quotes = GetQuotesLastPrice(positions.Select(x => x.ProjectedBalances.Symbol));
+
             foreach (var hold in positions)
             {
-                var symbol = Symbol.Create(hold.ProjectedBalances.Symbol, SecurityType.Equity, Market.USA);
+                var brokerageSymbol = hold.ProjectedBalances.Symbol;
```

Comment:
Should use the SymbolMapper to convert from brokerage symbol into lean symbol 👌

Reply:
Implemented `GetLeanSymbol()`